package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "puma-cli",
+  "version": "1.0.0",
+  "preferGlobal": true,
+  "bin": {
+    "puma-cli": "./src/puma-cli.js"
+  },
+  "description": "command line interface por jobs with PumaScript",
```

Comment:
Command line tool for PumaScript Framework

Reply:
Done.